.github/workflows/build-python-wheels.yml

```diff
@@ -51,7 +41,7 @@ jobs:
     runs-on: ubuntu-20.04
```

Comment:
Was this version supposed to be updated too? Or is it held back for e.g. GLIBC errors.

On that note, I just want a sanity check that none of these updates could introduce GLIBC errors.

Reply:
This line is used for building the source distribution, so it cannot have any effect on GLIBC errors.

On line 16, the one for building binaries, I updated it because at some point, it did not build python 3.10 and 3.11 and I thought that this was the problem. In fact, my mistake was somewhere else (`pyproject.toml`). So one could revert to 20.04 if you think it's better.

I have no idea of how we can deal with GLIBC problems: is it as for `numpy` by using the lowest version ? The only question is on line 16, and I let you decide what is the best ubuntu version to put there.

Reply:
I would prefer to leave these OS versions as they were before to avoid any issues. When we have made further progress on our own internal infrastructure, we will hopefully be a bit more flexible.

Ubuntu 20 has full support until 2025, and LTS until 2030, so unless there are missing features it is best to be conservative.

Reply:
> I would prefer to leave these OS versions as they were before to avoid any issues

OK: Ubuntu version for building wheels is back to 20.04

> Ubuntu 20 has full support until 2025, and LTS until 2030,

The question is rather: how long will GitHub keep a runner with this version. It looks they keep 2 versions, so 20.04 will likely disappear when 24.04 is released. That still gives us some time…

Reply:
Good point, it should last until Dec '24.